### PR TITLE
Updating Site Picker looks

### PR DESCRIPTION
**Pull request by kwonye**
Addresses #4349 

Updates site picker to new look on web.

Previous:

Now:

Needs sign-off from @apeatling before merging

### CODE REVIEW

**Comment by folletto** [comment]
Design wise, all seems good (and bonus point for a PR like this that consolidate the design), but two things:

1. The style of the button at the end of the list seems a bit odd to me. This isn't a blocker, but we need to do a design exploration for that.
2. That button on iOS does a different thing, which means that it shouldn't be using the same text used on .com. Until we are able to create a new site on iOS as well (which will require some redesign work), let's use a different copy, like "Add a WordPress" so it's clear it's not "new".

**Comment by apeatling** [comment]
In keeping with some of the iOS patterns in other apps I'm seeing, do you think the "Add" button be in the toolbar on the right next to search?

**Comment by folletto** [comment]
I think we discussed that a while ago... we opted to keep it at the bottom because it's not a strong pattern – it's stronger having just one icon there at the top. So ideally we should have search in-page (the usual scroll-down-a-bit-and-see-it) and probably add as an icon at the top. 

That said, it's a rather important piece, so I'd want more designers to chime in before implementing, and in the mean time I feel the above is already an improvement which we can polish and merge, waiting for the design. :)

**Comment by apeatling** [comment]
:+1:

File: WordPress/WordPressTest/ViewRelated/Blog/Style/WPStyleGuide+BlogTests.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        testCell = UITableViewCell(style: UITableViewCellStyle.Subtitle, reuseIdentifier: nil)
+        super.setUp()
+    }
+    
+    override func tearDown() {
+        super.tearDown()
+        testCell = nil
+    }
+    
+    func testConfigureTableViewBlogCellSetsFont() {
```
Comment (sendhil) [inline WordPress/WordPressTest/ViewRelated/Blog/Style/WPStyleGuide+BlogTests.swift]: Nice

**Comment by sendhil** [comment]
Code looks good to me. :shipit:

**Pull request merged**